cardano-api/src/Cardano/Api/Block.hs

```diff
@@ -251,6 +254,13 @@ data ChainPoint = ChainPointAtGenesis
                 | ChainPoint !SlotNo !(Hash BlockHeader)
   deriving (Eq, Show)
 
+instance ToJSON ChainPoint where
+  toJSON ChainPointAtGenesis = Aeson.String "Tip is currently at genesis block"
+  toJSON (ChainPoint slot headerHash) =
+    object [ "Slot Number" .= slot
+           , "Slot Header Hash" .= Text.decodeUtf8 (serialiseToRawBytesHex headerHash)
```

Comment:
It's unusual for keys to have spaces in them.  Is this for a specific reason?

Reply:
Nop just a bad habit, I'll simplify them.